site/src/components/LandingPage.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
-import landingStyles from '@site/src/components/css/LandingPage.module.css';
 import ItemGrid from '@site/src/components/ItemGrid.js';
 
 export default function LandingPage({itemList, Svg, h1Text, h2Text}) {
     return (
-        <div className={landingStyles.landing}>
-            <div className={landingStyles.left}>
-                <div className={landingStyles.title}>
+        <div className="landing-page">
+            <div className="left">
```

Comment:
`left` and `right` also don't really give any idea of where they're used or what they're for. Could this be made more specific? These don't exist anywhere else so they might as well be `landing-page-left` and `landing-page-right` so they're easier to find and modify later.